.dccache

```diff
@@ -1 +0,0 @@
ac3e21159a6ecb5bc6380d"],"/home/runner/work/plasma/plasma/packages/react/src/components/select/hoc/tests/SelectWithFilterReducers.spec.ts":[4264,1666966979258.6318,"6d607fe17b00713fbafcb06144bc0b4cb8c341fed20fdf2fa4c81e483ebbd38e"],"/home/runner/work/plasma/plasma/packages/react/src/components/select/hoc/tests/SelectWithInfiniteScroll.spec.tsx":[3537,1666966979258.6318,"e1c9d3d2f5ef5b441158d87a840e19b4135aacde60061642300b32ef1da0b56c"],"/home/runner/work/plasma/plasma/packages/react/src/components/select/hoc/tests/SingleSelectWithFilter.spec.tsx":[13804,1666966979258.6318,"20c184b51eb0bec1f227d84cbb338d606b035ade2758c691e35c80221a9bfe2d"],"/home/runner/work/plasma/plasma/packages/react/src/components/validation/components/tests/ValidationMessage.spec.tsx":[6735,1666966979270.6318,"48d5a8174bfc66b70f458f2b10fbcc06eda3d5464e8292f327fad055c79d0450"],"/home/runner/work/plasma/plasma/packages/react/src/components/validation/hoc/tests/WithDirtyCheckboxHOC.spec.tsx":[4168,1666966979270.6318,"efa446630293ae56905c4db1d23bcea362c797d23b43196a8c4d68154ce262e1"],"/home/runner/work/plasma/plasma/packages/react/src/components/validation/hoc/tests/WithDirtyInputHOC.spec.tsx":[4964,1666966979270.6318,"f24d0c1bb5f7c5716b1418b74e308375d049afe94ebd0c54fd19d4f785a17b79"],"/home/runner/work/plasma/plasma/packages/react/src/components/validation/hoc/tests/WithDirtyMultiSelectHOC.spec.tsx":[2568,1666966979270.6318,"32a34e92e7b23db611cbb98c55e10affd425122a7392d87700a1e3825be931df"],"/home/runner/work/plasma/plasma/packages/react/src/components/validation/hoc/tests/WithDirtySaveButtonHOC.spec.tsx":[9005,1666966979270.6318,"696cb637cc92b8e4fcfd8eb81f5769f111aa5f0d9f9d321e5f87c8c62358bd6b"],"/home/runner/work/plasma/plasma/packages/react/src/components/validation/hoc/tests/WithDirtySingleSelectHOC.spec.tsx":[3012,1666966979270.6318,"a14ef67c27233728f48b28157160d095d17aebd5e7ea630a28377d36d10f27f6"],"/home/runner/work/plasma/plasma/packages/react/src/components/validation/hoc/tests/WithDirtyStickyFooter.spec.tsx":[3418,1666966979270.6318,"d11382564376b149ad30c87c03321870c5e8c97d962ff4462e00b540b11f925c"],"/home/runner/work/plasma/plasma/packages/react/src/components/validation/hoc/tests/WithInitialValuesMultiSelectHOC.spec.tsx":[2953,1666966979270.6318,"cba9da1effbedbc6716a8a4a4157686809898fea38d30f427eca821997e015f2"],"/home/runner/work/plasma/plasma/packages/react/src/components/validation/hoc/tests/WithNonEmptySingleSelectHOC.spec.tsx":[1874,1666966979270.6318,"125b7df99593b8bf9bd0b39f9dc33aa519fa244abdf9bd94b383f0b1bbcbaf2a"],"/home/runner/work/plasma/plasma/packages/react/src/components/validation/hoc/tests/WithNonEmptyValueInputValidationHOC.spec.tsx":[2934,1666966979270.6318,"381aad124efbf6fa8958e6f3b29635415c1d264c2b2d3ae184c27172a47c2b43"],"/home/runner/work/plasma/plasma/packages/react/src/components/validation/hoc/tests/WithNonEmptyValueMultiSelectValidationHOC.spec.tsx":[1877,1666966979270.6318,"abc1298b9cd6f69bccd80dbf6ce7f00e309f6728f535c6a4ac879e3bf2bd1180"]}
```

Comment:
This is a cache file created by snyk when scanning the repo. I forgot to add it to gitignore so it got commited in the previous CD run. Here I am just deleting it to avoid polluting everyone